service/history/workflow/mutable_state_impl.go

```diff
@@ -8559,14 +8578,12 @@ func (ms *MutableStateImpl) reschedulePendingWorkflowTask(invalidatePendingTasks
 		return nil
 	}
 
-	if invalidatePendingTasks {
-		// Increase the stamp ("version") to invalidate the pending non-speculative WFT.
-		// We don't invalidate speculative WFTs because they are very latency sensitive.
-		ms.executionInfo.WorkflowTaskStamp += 1
+	// Increase the stamp ("version") to invalidate the pending non-speculative WFT.
+	// We don't invalidate speculative WFTs because they are very latency sensitive.
+	ms.executionInfo.WorkflowTaskStamp += 1
```

Comment:
let's set ms.workflowTaskUpdated = true since we are not using UpdateWorkflowTask() method here.